packages/common/src/types/utility.ts

```diff
@@ -1,6 +1,6 @@
 // This type ensures that the resulting value of a `keyof` is
 // a string.
 // eslint-disable-next-line @typescript-eslint/ban-types
-export type KeyOf<T extends object> = Extract<keyof T, string>;
+export type KeyOf<T extends object> = Extract<keyof T, string & keyof T>;
```

Comment:
I couldn't see any usages of `KeyOf` now. could it be removed?

Reply:
will do!